src/components/MediaTile.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+    <div class="MediaTile">
+        <b-img class="image" :src="imgSourceUri" :alt="title+' Thumbnail'" fluid rounded/>
```

Comment:
```suggestion
        <b-img class="image" :src="imgSourceUri" :alt="title+' Thumbnail'" fluid rounded></b-img>
```
Apparently "empty tag doesn't work in some browsers".

Reply:
Interesting...